Codeinwp/templates-patterns-collection
# PRF Changes

### Summary
- Changes plugin name;
- Deprecates the old `Page templates` tab inside the Template cloud inserter modal;
- Now, only users with a valid library license can see the block/elementor/beaver functionalities;
- Valid licenses are either Template Cloud dedicated ones, or Neve licenses from tier 3;
- Migrates old users to the new experience, depending on the active license, and conditioned by the templates they previously had saved in the cloud according to [this comment here](https://github.com/Codeinwp/templates-cloud/issues/91#issuecomment-2467982241).


### Will affect visual aspect of the product

YES (but only on the backend)


### Test instructions


- Follow the flows mentioned in this comment and confirm that the plugin acts accordingly: https://github.com/Codeinwp/templates-cloud/issues/91#issuecomment-2467982241
- Except from what is mentioned there, when the user doesn't have neve, we still show the import buttons, except that when the user tries importing a starter site, they will be prompted to install the theme as well. 

![Arc 2024-11-25 15 44 30](https://github.com/user-attachments/assets/b85162e9-ac21-44ce-a5b3-79cefce15486)


## Check before Pull Request is ready:



Closes Codeinwp/templates-cloud#91.
